### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by pranav-bhatt**
Updated the README.md file:
* Updated Drogue IoT Cloud description
* Added a protocol endpoint support section
* Added a "Useful Links" section containing links to documentation and the Drogue IoT blog
* Added a community section containing links to the IRC room, bi-weekly community call calendar, Drogue IoT forum, YouTube channel and Twitter handle 
Signed-off-by: Pranav <adpranavb2000@gmail.com>

### CODE REVIEW

**Review by lulf — APPROVED** [review]
I like the changes, great to have the community links in here.

File: README.md [hunk 1]
```diff
@@ -37,6 +43,19 @@ minikube tunnel # in a separate terminal, as it keeps running
...
+* [Documentation](https://book.drogue.io/drogue-cloud/dev/index.html) 
+* [Drogue IoT Blog: Articles that talk about the design,  usecases and project updates](https://blog.drogue.io/)
+
 ## Contributing
 
 See the document [CONTRIBUTING.md](CONTRIBUTING.md).
+
+## Community
+
+* [Drogue IoT IRC Chat Room](https://matrix.to/#/#drogue-iot:matrix.org)
```
Comment (ctron) [inline README.md:57]:
```suggestion
* [Drogue IoT Matrix Chat Room](https://matrix.to/#/#drogue-iot:matrix.org)
```

Comment (ctron) [reply README.md:57]: Technically it is "matrix", not "IRC".

**Review by ctron — CHANGES_REQUESTED** [review]
Thanks for the PR. This is a great improvement indeed.

I would only ask to replace "IRC" and maybe add a note to CoAP that this might come in the near future. Otherwise we could add a bunch of other protocols that we don't support, but I am not sure there is a benefit in that. CoAP however is different as we don't support this, but there are plans to change this soon.

File: README.md (hunk 1 above)
Comment (pranav-bhatt) [reply README.md]: Thanks for that!

Comment (pranav-bhatt) [reply README.md]: So would changing the 'x' into a "Coming Soon" suffice?

**Review by ctron — APPROVED** [review]
Thanks for the PR!

**Pull request merged**